manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -82,22 +82,50 @@ spec:
     rules:
     - alert: UsingDeprecatedAPIAppsV1Beta1
       annotations:
-        message: A client in the cluster is using deprecated apps/v1beta1 API that will be removed soon.
+        summary: |
+          A client is using a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version
+        message: |
+          A client in the cluster has used a deprecated {{"{{"}} $labels.group {{"}}"}}/{{"{{"}} $labels.version {{"}}"}} API version of the {{"{{"}} $labels.resource {{"}}"}} resource in the last 24 hours.
+          This API version will be removed soon; please migrate to the current version of this API.
       expr: |
-        apiserver_request_count{group="apps",version="v1beta1"}
+        increase(
+          apiserver_request_total{
+            group="apps",
+            version="v1beta1",
+          }[24h]
+        ) > 0
       labels:
         severity: warning
```

Comment:
Should this now change to info, to be in line with the master PR (https://github.com/openshift/cluster-kube-apiserver-operator/pull/742/files#diff-9cc7eba97b17dc12b44afb453dc9e69cL126) ?